packages/asset-swapper/src/constants.ts

```diff
@@ -1,55 +1,65 @@
-import { SignedOrder } from '@0x/types';
 import { BigNumber } from '@0x/utils';
 
 import {
-    ForwarderSwapQuoteExecutionOpts,
-    ForwarderSwapQuoteGetOutputOpts,
-    OrdersAndFillableAmounts,
+    ForwarderExtensionContractOpts,
+    OrderPrunerOpts,
+    OrderPrunerPermittedFeeTypes,
+    SwapQuoteExecutionOpts,
+    SwapQuoteGetOutputOpts,
     SwapQuoteRequestOpts,
     SwapQuoterOpts,
 } from './types';
 
 const NULL_BYTES = '0x';
+const NULL_ERC20_ASSET_DATA = '0xf47261b00000000000000000000000000000000000000000000000000000000000000000';
 const NULL_ADDRESS = '0x0000000000000000000000000000000000000000';
 const MAINNET_NETWORK_ID = 1;
 const ONE_SECOND_MS = 1000;
 const DEFAULT_PER_PAGE = 1000;
+const PROTOCOL_FEE_MULTIPLIER = 150000;
 
-const DEFAULT_SWAP_QUOTER_OPTS: SwapQuoterOpts = {
-    networkId: MAINNET_NETWORK_ID,
-    orderRefreshIntervalMs: 10000, // 10 seconds
+const DEFAULT_ORDER_PRUNER_OPTS: OrderPrunerOpts = {
     expiryBufferMs: 120000, // 2 minutes
+    permittedOrderFeeTypes: new Set<OrderPrunerPermittedFeeTypes>([
+        OrderPrunerPermittedFeeTypes.NoFees,
+        OrderPrunerPermittedFeeTypes.MakerDenominatedTakerFee,
+    ]), // Default asset-swapper for CFL oriented fee types
+};
```

Comment:
I am fine with this namely because developers shouldn't be overriding this option often, if not at all. The only case these orderFeeTypes will be overridden are for non CFL use cases or non ERC20 assets; with that, the only case that these are overridden would be cases like Instant.